pkg/loadbalancer/tunnel.go

```diff
@@ -39,9 +39,9 @@ func (t *tunnelManager) setupTunnels(containerName string) error {
 	}
 
 	klog.V(0).Infof("setting IPv4 address %s associated to container %s", ipv4, containerName)
-	err = AddIPToLocalInterface(ipv4)
+	output, err := AddIPToLocalInterface(ipv4)
 	if err != nil {
-		return err
+		klog.V(4).Infof("error adding IP to local interface: %v - %s", err, output)
```

Comment:
wrap the error here, I think this `tunnel` is ok to return the error ... I find better if we swallow it here by logging it at normal level, as the tunnel is an addon for certain environments https://github.com/kubernetes-sigs/cloud-provider-kind/blob/defae812a313ae5551727869139734903852da36/pkg/loadbalancer/server.go#L120-L126

Reply:
pushed - take a look at the edits